R/onLoad.R

```diff
@@ -0,0 +1,5 @@
+.onLoad <- function(libname, pkgname) {
+  if (is.null(getOption("rsconnect.max.bundle.size"))) {
+    options(rsconnect.max.bundle.size = 1048576000)
```

Comment:
Thanks! Is this exposed anywhere? I don't think the package today knows anything about service plans.